src/App.vue

```diff
@@ -35,9 +35,13 @@
         </ul>
         <!--Search Input -->
         <div class="search-container">
-          <input class="form-control me-2 searchInput" type="search" v-model="title"
```

Comment:
Looks good! We can give album or artist or track name in the title bar to search..It makes our search easy